### PR TITLE
OCPBUGS-19820: add link to dracut man page

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-19820

Link to docs preview: https://66981--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal#installation-user-infra-machines-routing-bonding_installing-bare-metal

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-user-infra-machines-static-network.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following information provides examples for configuring networking and bondi
 Ordering is important when adding the kernel arguments: `ip=`, `nameserver=`, and then `bond=`.
 ====
 
-The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the `dracut.cmdline` manual page.
+The networking options are passed to the `dracut` tool during system boot. For more information about the networking options supported by `dracut`, see the link:https://www.man7.org/linux/man-pages/man7/dracut.cmdline.7.html[`dracut.cmdline` manual page].
```
Comment (bergerhoffer) [inline modules/installation-user-infra-machines-static-network.adoc:64]: So it's not official guidance yet, but we've been working on an update to the SSG regarding referencing man pages, and one of the things it says is to not link to them.

Again, nothing official yet so I don't think you need to change it, just wanted to give a heads up!

**Review by bergerhoffer — COMMENTED** [review]
Just a comment on adding the link!

**Comment by jldohmann** [comment]
@mike-nguyen PTAL for QE ack, just adds a link for more info on `dracut`, thanks!

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**